basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,182 @@
 const helpers = require('./project-1');
 
 // start testing!
+// are equal
+describe( 'areEqual(x, y)', function ()
+{
+    it( 'should return true if the arguments are equal', function ()
+    {
+        expect( helpers.areEqual( 15, 15 ) ).toBe( true );
+        expect( helpers.areEqual( 90, 50 ) ).toBe( false );
+        expect( helpers.areEqual( 'test', 'test' ) ).toBe( true );
+    } );
+} );
+// are same length
+describe( 'areSameLength(str1, str2)', function ()
+{
+    it( 'should return true if the arguments have the same length', function ()
+    {
+        expect( helpers.areSameLength( 'hi', 'there' ) ).toBe( false );
+        expect( helpers.areSameLength( 'javascript', 'bumfuzzled' ) ).toBe( true );
+    } );
+} );
+//lessthan ninety
+describe('lessThanNinety(num)', function() {
```

Comment:
What type of value does this function return? Perhaps check for that as well.
Consider checking if [70] returns anything at all.
Consider checking a string of a number like '100' or '70'